demonstrations/tutorial_initial_state_preparation.py

```diff
@@ -227,7 +225,7 @@
 
 # generate the molecular Hamiltonian for H3+
 symbols = ["H", "H", "H"]
-geometry = np.array([[0, 0, 0], [0, 0, R/0.529], [0, 0, 2*R/0.529]])
+geometry = np.array([[0, 0, 0], [0, 0, R / 0.529], [0, 0, 2 * R / 0.529]])

```

Comment:
You should now be able to use this 🤔 
```suggestion
molecule = qchem.Molecule(symbols, geometry, charge=1, unit="angstrom")
```